README.md

```diff
@@ -23,6 +23,8 @@ You can view them at:
 [GOVERNANCE.md](./GOVERNANCE.md)  
 [CONTRIBUTING.md](./CONTRIBUTING.md)
 
+The Evangelism WG is overseen by the [Node.js Community Committee](https://github.com/nodejs/community-committee).
```

Comment:
@Trott Agreed on that - I went with this because it was pulled from the Node.js Collection Team: https://github.com/nodejs/nodejs-collection#about-this-team

That said, it's a team and this is a WG. I thought about changing it, but didn't know what the best phrasing would be - so I went with the phrasing in the repo above. Will change it to `chartered by` 👍